index.d.ts

```diff
@@ -139,13 +144,7 @@ declare module 'plaid' {
   }
 
   interface InstitutionWithContactData extends Institution {
-    addresses: Array<{
```

Comment:
@tanatplaid we can actually remove `InstitutionWithContactData` - it is no longer possible to get this from the API.

Reply:
⚡️